src/Steps/Attachment1Graphics.tsx

```diff
@@ -118,6 +118,50 @@ export const Attachment1Graphics = () => {
     dispatch({ type: "MERGE_OPTION", payload });
   };
 
+  const handleCustom = (e: React.ChangeEvent<HTMLInputElement>) => {
+    // Bit hacky here using the any type, but it allows us to dynamically name the payload attributes
+    let payload: any = {};
+    const payloadName = e.target.id.slice(0, e.target.id.lastIndexOf("_"));
+    const index = e.target.id.substring(e.target.id.lastIndexOf("_") + 1);
+
+    payload.graphics = [
+      ...globalState.wizardOptions[globalState.tmcrIndex]?.graphics,
+    ];
+    payload.graphics[index][payloadName] = e.target.value;
+
+    dispatch({ type: "MERGE_OPTION", payload });
+  };
+
+  const addRow = () => {
+    let payload: any = {};
+    const newEntry = {
+      title: "",
+    };
+
+    if (!globalState.wizardOptions[globalState.tmcrIndex].graphics) {
+      payload.graphics = [];
+    } else {
+      payload.graphics = [
+        ...globalState.wizardOptions[globalState.tmcrIndex].graphics,
+      ];
+    }
+    payload.graphics.push(newEntry);
+    payload.graphics_none = false;
+
+    dispatch({ type: "MERGE_OPTION", payload });
+  };
+
+  const deleteRow = (index: number) => {
+    let graphics = [
+      ...globalState.wizardOptions[globalState.tmcrIndex].graphics,
+    ];
+    graphics.splice(index, index + 1);
```

Comment:
This should be the below -- otherwise you will delete more than 1 if not selecting the first index
```suggestion
    graphics.splice(index, 1);
```